sdks/java/io/hcatalog/pom.xml

```diff
@@ -35,6 +35,118 @@
     <apache.commons.version>2.5</apache.commons.version>
   </properties>
 
+  <profiles>
+    <!--
+           This profile invokes PerfKitBenchmarker, which does benchmarking of
+           the IO ITs. The arguments passed to it allow it to invoke mvn again
+           with the desired benchmark.
+
+           To invoke this, run:
+
+           mvn verify -Dio-it-suite -pl sdks/java/io/file-based-io-tests
```

Comment:
@iemejia thanks for this question, because it gives a feedback meaning that we should popularise and describe the IOITs more!

The test you're currently reviewing is specific: currently one can run it only if he/she has all existing infrastructure set up. We currently (locally) use google dataproc and special [init action](https://github.com/GoogleCloudPlatform/dataproc-initialization-actions/tree/master/hive-hcatalog) to setup needed infrastructure. 
Please consider this a work in progress: this is yet do be decided if we provide dedicated kubernetes setup for this test (the way we did in other tests) or setup this standalone dataproc infrastructure and use it on jenkins.

There is a website, but IMO it needs solid improvement. This is also a pending task, but it's to be done after the gradle migration (which I'm finishing it right now). It might be a good idea to describe current HCatalog infrastracture setup there too. 

@kkucharc let's add a README.md file with infrastructure description for this test only, at least for now, given that there's no mention about the exceptional setup and we still didn't decide which option is best. I think this is the best and quickest solution. @iemejia do you agree?

Reply:
@iemejia Sorry for mistake, I will repair it.

Yes, we were able to run it on dataproc infrastructure. It was much more challenging to run it on docker and finally we failed due to network problems. Because test was working, we decided to create PR and postpone docker/k8s/setup task.
@lgajowy I agree - I will add README. Maybe new JIRA issue as well?